docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks-isv.md

```diff
@@ -0,0 +1,526 @@
+---
+id: eks-isv
+title: Replicated PV Mayastor Installation on Amazon Elastic Kubernetes Service with Instance Store Volumes
+keywords:
+ - Replicated PV Mayastor Installation on Amazon Elastic Kubernetes Service with Instance Store Volumes
+ - Replicated PV Mayastor Installation on EKS with Instance Store Volumes
+ - Replicated PV Mayastor - Platform Support
+ - Platform Support
+ - Elastic Kubernetes Service with Instance Store Volumes
+ - EKS
+description: This section explains about the Platform Support for Replicated PV Mayastor.
+---
+# Replicated PV Mayastor Installation on Amazon Elastic Kubernetes Service with Instance Store Volumes
+
+This document provides instructions for installing Replicated PV Mayastor on Amazon Elastic Kubernetes Service (EKS) with Instance Store Volumes. Replicated PV Mayastor is designed to work with Amazon EKS with Instance Store Volumes. It provides high-performance storage for stateful applications running in a Kubernetes environment. 
+
+Using OpenEBS Replicated PV Mayastor in EKS with Instance store volumes addresses many of the limitations associated with the ephemeral nature of local SSDs by introducing a layer of persistent storage management. Here's how OpenEBS helps mitigate these limitations:
+
+**Data Persistence through Replication**
+
+- OpenEBS abstracts the underlying storage (Instance store volumes) into a set of persistent volumes. Even if local SSDs are inherently ephemeral, OpenEBS can ensure data persistence by replicating the data across multiple worker nodes.
+
+- For instance, using OpenEBS with replication (e.g., 2 or 3 replicas), ensures that even if one node fails or is terminated, the data exists on other nodes, avoiding data loss.
```

Comment:
Is it e.g. or eg. ?

Reply:
It's "e.g." @avishnu . Or we can write it as "Example"